### PR TITLE
zh-CN: converted translation to po file format

### PR DESCRIPTION
**Pull request by jmnie**
- Converted translation files to po format

- Added translation for faq.po and COVID-19.po

### CODE REVIEW

File: Localization/zh-CN/covid-19.po [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+msgstr "1. 分享计算资源："
+
+msgid "[Downloading Folding@home](/start-folding/) and helping us run simulations is the primary way to contribute. These calculations are enormous and every little bit helps! Each simulation you run is like buying a lottery ticket. The more tickets we buy, the better our chances of hitting the jackpot. Usually, your computer will never be idle, but we’ve had such an enthusiastic response to our COVID-19 work that you will see some intermittent downtime as we sprint to setup more simulations. Please be patient with us! There is a lot of valuable science to be done, and we’re getting it running as quickly as we can."
+msgstr "[下载 Foling@home 模拟软件](/start-folding/)并帮助我们运行模拟是主要的贡献方式。模拟运算非常复杂并且每一点计算都可以帮到我们！您进行的每次模拟都像买彩票一样。我们购买的门票越多，我们赢得大奖的机会就越大。通常情况下，您的计算机永远不会处于闲置状态，但是我们对COVID-19的工作表现出了极大的热情，以至于我们加紧展开更多模拟时，您会发现间歇性的停机时间。请耐心等待我们！有许多有价值的科学实验要做，而且我们正在尽可能快地使它运行。"
+
+msgid "Don’t miss our client FAQ. [HERE](https://foldingforum.org/viewtopic.php?f=24&t=32463)"
+msgstr "不要错我们的常见问题解答。[点击这里](https://foldingforum.org/viewtopic.php?f=24&t=32463)"
+
+msgid "To focus on Covid19 research set your category to any."
+msgstr "为了专注于Covid19研究，请将您的类别设置为任意。"
```
Comment (Tar-Minyatur) [inline Localization/zh-CN/covid-19.po]: There was an update to the file very recently. Feel free to add that to this PR, if you want to get that change in here as well. Can also be added later of course.

**Review by Tar-Minyatur — DISMISSED** [review]
Looks good. Thanks for the contribution! :)
(There is one comment...doesn't have to be solved in this PR though.)

File: Localization/zh-CN/covid-19.po (hunk 1 above)
Comment (jmnie) [reply Localization/zh-CN/covid-19.po]: Thanks! May you please request changes in this PR so I can edit it (I not familiar with the functionality of pull request and do not know other methods to edit the PR )? Thank you.

**Review by kn1cht — CHANGES_REQUESTED** [review]
> Thanks! May you please request changes in this PR so I can edit it (I not familiar with the functionality of pull request and do not know other methods to edit the PR )? Thank you.

**Review by kn1cht — APPROVED** [review]

File: Localization/zh-CN/covid-19.po (hunk 1 above)
Comment (kn1cht) [reply Localization/zh-CN/covid-19.po]: Sorry, I didn't noticed that you had already pushed this changes. I have already approved about my review.

**Pull request merged**